library/alloc/src/vec/mod.rs

```diff
@@ -828,6 +833,7 @@ impl<T, A: Allocator> Vec<T, A> {
     /// }
     /// # process_data(&[1, 2, 3]).expect("why is the test harness OOMing on 12 bytes?");
     /// ```
+    #[doc(alias = "realloc")]
```

Comment:
Even though this `realloc` alias makes sense, having so many of it is kinda killing its purpose, no?

Reply:
Rust has no canonical "realloc" function; instead it's split up between various different methods which perform specific kinds of reallocations. Someone searching for "realloc" may not be aware of this, and sharing the various ways in which vectors can be reallocated may be helpful.

Conversely an experienced Rust programmer may not recall the exact name of a specific reallocation they're trying to perform (or may want to validate whether a vec method indeed reallocates) and searching for "realloc" may provide them with an answer.

Reply:
Fine by me, let's give it a try like this then!